### ISSUE TITLE
Error: Locations: No module for https://unpkg.com/html5-qrcode

### ISSUE THREAD

**Issue by gh-pap** [Afinas-nl/qr-bar-code-scanner-dialog#4]
tl;dr web app using qr_bar_code_scanner_dialog has this error: "Locations: No module for https://unpkg.com/html5-qrcode".

Longer version:
Thanks so much for putting this package together. There's a definite need of a barcode scanner that works reliably for PWAs - I've been trying several of them over the last 24hrs and they all had varying issues so I was excited to try qr_bar_code_scanner_dialog. Everything seemed promising but once I was at the video view my debug console showed 6 "Locations: No module for https://unpkg.com/html5-qrcode" errors and I wasn't able to get a successful scan. I was on stable Flutter 3.3.4 so, as an experiment, I downgraded to stable 3.3.3 and tried again but I have the same issue.  Is anyone else seeing this issue? I've double checked the install instructions (" Web Integration -- No need to update anything, the plugin appends the HTML contents to the DOM") -- the install is so straightforward (one of the great features of this package) I don't think I've missed anything, but maybe I have?

**Comment by wsk321** [comment]
Brother, have you solved this problem? I also encountered it
